test/common/testlib.py

```diff
@@ -1215,13 +1215,14 @@ def check_axe(self, label=None, suffix=""):
         will be appended to the file name, which is useful if you call this
         more than once within one test.
         """
-        if not checkRunAxe():
-            return
-        # HACK: We cannot test axe on Firefox since `axe.run()` returns promise
-        # and Firefox CDP cannot wait for it to resolve
+
+        # Only test Axe on chromium browsers
         if self.browser.cdp.browser == "firefox":
```

Comment:
Then let's actually make it do that with `!= "chromium`", just in case we get webkit or whatnot.